src/main/scripts/cbas

```diff
@@ -29,6 +29,9 @@ def get_jump_url(config):
     Return the jumphost url with HTTP scheme if we're in testing mode,
     HTTPS otherwise.
     """
+    if "://" in config.jump_host:
```

Comment:
Yeah, I was never too happy with this solution. Maybe we can even get rid of the one below, because it is an artificial coupling and simply use a url with protocol in the tests?